packages/core/src/command/patient-import/patient-import-local.ts

```diff
@@ -0,0 +1,143 @@
+import { executeAsynchronously } from "../../util/concurrency";
+import { createJobRecord } from "./commands/create-job-record";
+import { validateAndParsePatientImportCsvFromS3 } from "./commands/validate-and-parse-import";
+import { checkPatientRecord } from "./commands/check-patient-record";
+import { creatOrUpdatePatientRecord } from "./commands/create-or-update-patient-record";
+import { startDocumentQuery } from "./commands/start-document-query";
+import { createPatient } from "./commands/create-patient";
+import { startPatientQuery } from "./commands/start-patient-query";
+import {
+  PatientImportHandler,
+  StartPatientImportRequest,
+  ProcessPatientImportRequest,
+  ProcessPatientCreateRequest,
+  ProcessPatientQueryRequest,
+} from "./patient-import";
+import { createPatientPayload } from "./patient-import-shared";
+
+export class PatientImportHandlerLocal implements PatientImportHandler {
+  constructor(private readonly patientImportBucket: string) {}
+
+  async startPatientImport({
+    cxId,
+    facilityId,
+    jobId,
+    rerunPdOnNewDemographics = true,
+    dryrun = false,
+  }: StartPatientImportRequest): Promise<void> {
+    const jobStartedAt = new Date().toISOString();
+    const processPatientImportRequest: ProcessPatientImportRequest = {
+      cxId,
+      facilityId,
+      jobId,
+      jobStartedAt,
+      s3BucketName: this.patientImportBucket,
+      processPatientCreateQueue: "local",
+      rerunPdOnNewDemographics,
+      dryrun,
+    };
+    const boundProcessPatientImport = this.processPatientImport.bind(this);
+    await boundProcessPatientImport(processPatientImportRequest);
+  }
+
+  async processPatientImport({
+    cxId,
+    facilityId,
+    jobId,
+    jobStartedAt,
+    s3BucketName,
+    rerunPdOnNewDemographics,
+    dryrun,
+  }: ProcessPatientImportRequest): Promise<void> {
+    await createJobRecord({
+      cxId,
+      jobId,
+      data: { jobStartedAt },
+      s3BucketName,
+    });
+    const patients = await validateAndParsePatientImportCsvFromS3({
+      cxId,
+      jobId,
+      s3BucketName,
+    });
+    if (dryrun) return;
```

Comment:
Can we log we're running w/ dryrun and not continuing, please?

Reply:
Done.